compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/DatePicker.kt

```diff
@@ -830,7 +831,7 @@ value class DisplayMode internal constructor(internal val value: Int) {
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Stable
-internal class StateData constructor(
+internal class StateData(
```

Comment:
Also, we don't refactor code in commonMain, even if it can be refactored, to avoid merge conflicts. If it is needed to be refactored, we should do it in AOSP.

Reply:
Right, I forgot about that.